src/helpers/__tests__/helpers.test.ts

```diff
@@ -3,132 +3,132 @@
+    ];
+
+    const result = helpers.noopTranslate(key, value, components);
+
+    expect(result).toBe(`t(${key}, ${value}, ${components})`);
+  });
+
+  it('should handle array keys and values', () => {
+    const key = ['key1', 'key2'];
+    const value = ['value1', 'value2'];
+
+    const result = helpers.noopTranslate(key, value, []);
 
-it('should return AuthType.UsernameAndPassword when credential has username and password', () => {
-  const mockCredential: CredentialType = {
-    id: 'mockId',
-    name: 'mockName',
-    created_at: new Date(),
-    updated_at: new Date(),
-    cred_type: 'mockCredType',
-    username: 'testUser',
-    password: 'testPassword',
-    ssh_keyfile: '',
-    auth_token: '',
-    ssh_passphrase: '',
-    become_method: '',
-    become_user: '',
-    become_password: '',
-    sources: []
-  };
-  const result = getAuthType(mockCredential);
-  expect(result).toBe(authType.UsernameAndPassword);
+    expect(result).toBe(`t([key1,key2], [value1,value2])`);
+  });
+
+  it('should handle empty components', () => {
+    const key = 'testKey';
+    const value = 'testValue';
+
+    const result = helpers.noopTranslate(key, value, []);
+
+    expect(result).toBe(`t(${key}, ${value})`);
+  });
 });
 
-it('should return AuthType.Token when credential has auth_token', () => {
-  const mockCredential: CredentialType = {
-    id: 'mockId',
-    name: 'mockName',
-    created_at: new Date(),
-    updated_at: new Date(),
-    cred_type: 'mockCredType',
-    username: '',
-    password: '',
-    ssh_keyfile: '',
-    auth_token: 'mockToken',
-    ssh_passphrase: '',
-    become_method: '',
-    become_user: '',
-    become_password: '',
-    sources: []
-  };
-  const result = getAuthType(mockCredential);
-  expect(result).toBe(authType.Token);
+describe('formatDate', () => {
+  it('should format the date correctly in UTC', () => {
+    const date = new Date('2022-01-01T12:00:00Z');
+    const formattedDate = helpers.formatDate(date);
+
+    expect(formattedDate).toEqual('01 January 2022, 12:00 PM UTC');
+  });
 });
 
-it('should return AuthType.SSHKeyFile when credential has ssh_keyfile', () => {
-  const mockCredential: CredentialType = {
-    id: 'mockId',
-    name: 'mockName',
-    created_at: new Date(),
-    updated_at: new Date(),
-    cred_type: 'mockCredType',
-    username: '',
-    password: '',
-    ssh_keyfile: 'mockSSH',
-    auth_token: '',
-    ssh_passphrase: '',
-    become_method: '',
-    become_user: '',
-    become_password: '',
-    sources: []
-  };
-  const result = getAuthType(mockCredential);
-  expect(result).toBe(authType.SSHKeyFile);
+it('should normalize total values', () => {
+  expect(helpers.normalizeTotal({ count: 142 }, true, undefined)).toEqual(42);
+  expect(helpers.normalizeTotal({}, true, undefined)).toEqual(0);
+  expect(helpers.normalizeTotal({ count: 142 }, false, undefined)).toEqual(142);
+  expect(helpers.normalizeTotal({ count: 142 }, true, 50)).toEqual(42);
```

Comment:
required: I missed all of the string `toEqual`s here too, apologies ugh, mostly since helped write that example lol, let's be consistent and use `toBe` on all of these instead

explanation is here [`toBe uses Object.is to test exact equality. If you want to check the value of an object, use toEqual instead:`](https://jestjs.io/docs/using-matchers)